City-of-Helsinki/helsinki-paatokset
# PP-281: add news item + use its template in imported articles

Enable helfi_news_item -module and use news item template for imported articles.

To test:
- Checkout branch. Run `make drush-cr; make drush-cim` (or `make new`)
- Run `drush mim paatokset_news --update` inside the container.
- Check that any imported article content exists (here)[https://helsinki-paatokset.docker.so/fi/admin/content?title=&type=imported_article&status=All&langcode=All]
- View an imported article page. See that it is visually identical to https://nginx-strategia-talous-test.agw.arodevtest.hel.fi/fi/test-paatoksenteko-ja-hallinto/uutinen-esimerkki-lyhyt-otsikko
